src/extension/background-script/actions/allowances/__tests__/enable.test.ts

```diff
@@ -4,7 +4,7 @@ import state from "~/extension/background-script/state";
 import { allowanceFixture } from "~/fixtures/allowances";
 import type { DbAllowance, MessageAllowanceEnable, Sender } from "~/types";
 
-import enableAllowance from "../enable";
+import enableAllowance from "../../webln/enable";
```

Comment:
this seems wrong related to the filename. Now do we need tests for each provider?

Reply:
what shall be do here?

Reply:
this shall be not in allowance anymore. separate file in each provider?